cmd/frontend/internal/httpapi/webhookhandlers/handle_repo_authz_event.go

```diff
@@ -0,0 +1,55 @@
+package webhookhandlers
+
+import (
+	"context"
+	"fmt"
+	gh "github.com/google/go-github/v28/github"
+	"github.com/inconshreveable/log15"
+	"github.com/sourcegraph/sourcegraph/cmd/frontend/backend"
+	"github.com/sourcegraph/sourcegraph/cmd/repo-updater/repos"
+	"github.com/sourcegraph/sourcegraph/internal/actor"
+	"github.com/sourcegraph/sourcegraph/internal/api"
+	"github.com/sourcegraph/sourcegraph/internal/conf"
+	"github.com/sourcegraph/sourcegraph/internal/repoupdater"
+	"github.com/sourcegraph/sourcegraph/internal/repoupdater/protocol"
+)
+
+// handleGithubRepoAuthzEvent handles any github event containing a repository field, and enqueues the contained
+// repo for permissions synchronisation
+func handleGitHubRepoAuthzEvent(ctx context.Context, extSvc *repos.ExternalService, payload interface{}) error {
+	if !conf.Get().ExperimentalFeatures.EnablePermissionsWebhooks {
```

Comment:
Just a sanity check: what happens if there is no ExperimentalFeatures configed at all in the site config, would this panic?


```suggestion
	if !conf.ExperimentalFeatures().ExperimentalFeatures.EnablePermissionsWebhooks {
```


https://github.com/sourcegraph/sourcegraph/blob/1d9e6f86d61c66e7c45e84374574850e29953d95/internal/conf/computed.go#L291-L296